ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineForkchoiceUpdated.java

```diff
@@ -162,7 +162,8 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
                     newHead,
                     payloadAttributes.getTimestamp(),
                     payloadAttributes.getPrevRandao(),
-                    payloadAttributes.getSuggestedFeeRecipient()));
+                    payloadAttributes.getSuggestedFeeRecipient(),
+                    Optional.empty()));
```

Comment:
nit: Could pass in withdrawals at this point, or could keep in separate PR?

After moving the validation into this class, there's no refactor necessary, should just be a case of mapping the payloadAttributes.withdrawals -> List<Withdraw>...e.g. WithdrawalParameter.toDomain()

Reply:
done

Reply:
done